lib/glific/contacts.ex

```diff
@@ -312,6 +315,12 @@ defmodule Glific.Contacts do
     Partners.organization(contact.organization_id).contact_id == contact.id
   end
 
+  @doc """
+  Checks if the contact is simulator
+  """
+  @spec is_simulator?(Contact.t()) :: boolean()
```

Comment:
Why are we creating a duplicate function here?  If not can you check how we are checking the simulator contact before this PR.

Reply:
made requested changes